src/components/2-Monitoring-Mobile/MatrixExplainerTime.vue

```diff
@@ -955,10 +955,22 @@ export default {
 
 <style lang="scss" scoped>
 
-$backgroundCharcoal: #171717;
+// IMPORT COLORS
+$backgroundCharcoal: #202226; //#1E1F23 #26282b 
+$boxCharcoal: #171717; //#28292D #1a1b1f
+$offWhite: #F1F1F1;
+$plasmaYellow: #FAB62F;
+$plasmaPink: #BE3D7D;
+$plasmaPurple: #62039A;
+$plasmaBlue: #142167;
+$darkBlue: #0F2237;
+$footerBlue: #00264C;
+$mediumBlue: #5191bd; //#63B1E6 #579ecf
+$lightBlue: #AAD1EC;
+$grayBlue: #4F5C67;
```

Comment:
hmm I wonder if there's a way to declare these with a global scope so they don't need to be repeated. 🤷🏼‍♂️

Reply:
I think there is. They seemed to carry through from App.vue where Ellen had defined them in most cases, but not always? I think maybe not in these imported components? I decided to just define the colors everywhere in case.